portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> messages = new ArrayList<String>();
+  
+  public DataServlet () {
+    messages.add("The sun is out today!");
+    messages.add("I just got new curtains.");
+    messages.add("Woo hoo!");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello There!");
+    String json = convertToJsonUsingGson(messages);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+ 
+  private String convertToJsonUsingGson(ArrayList<String> messagesParam) {
```

Comment:
Since the underlying method of converting the strings to JSON may change, I would change this method name to describe what the function does rather than how it does it